tests/NLog.UnitTests/Targets/FileTargetTests.cs

```diff
@@ -3269,6 +3269,36 @@ public void TestFilenameCleanup()
             Assert.Equal(expectedFileName, path);
         }
 
+        [Fact]
+        public void TestCalculateNextWeekday()
+        {
+            DateTime today = new DateTime(2017, 03, 02, 15, 27, 34, 651);
+            DateTime expected = new DateTime(2017, 03, 05, 15, 27, 34, 651);
+            FileTarget fileTarget = new FileTarget();
+            DateTime actual = fileTarget.CalculateNextWeekday(today, DayOfWeek.Sunday);
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
```

Comment:
would be nice to combine the unit tests to one test, with `[InlineData]` , see http://www.codinginstinct.com/2008/10/funky-xunit-theory.html